src/signals/incident-management/containers/IncidentDetail/components/History/index.js

```diff
@@ -1,27 +1,55 @@
 import React from 'react';
-import { historyType } from 'shared/types';
 
+import styled, { css } from '@datapunt/asc-core'
+import { themeColor, Heading, styles } from '@datapunt/asc-ui'
 import { string2date, string2time } from 'shared/services/string-parser/string-parser';
+import { historyType } from 'shared/types';
+
+const rowStyles = css`
+  display: flex;
+  flex-wrap: wrap;
+  margin-right: -15px;
+  margin-left: -15px;
+`;
+
+const StyledHistoryListItem = styled.div`
+  margin-bottom: 16px;
+  line-height: 22px;
+  ${rowStyles}
```

Comment:
@janjaap Column behaves very different from the current used `col` classes.
It uses different breakpoints and column counts. It would require to overwrite theme options to get the same results